schema/fmi3AttributeGroups.xsd

```diff
@@ -102,14 +102,13 @@ ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
                 <xs:restriction base="xs:normalizedString">
                     <xs:enumeration value="constant"/>
                     <xs:enumeration value="fixed"/>
-                    <xs:enumeration value="calculated"/>
                     <xs:enumeration value="tunable"/>
-                    <xs:enumeration value="changing"/>
```

Comment:
Does the removal of changing originate from my PR?
It that case I belive it was a mistake

Reply:
I added that back into your initial PR - @t-sommer must rebase his proposal, I guess...